docs/resources/security_policy.md

```diff
@@ -24,6 +24,7 @@ resource "xray_security_policy" "min_severity" {
 
     criteria {
       min_severity = "High"
+      fix_version_dependant = false
```

Comment:
HCL alignment?

Reply:
fixed, this one is generated from `examples/resources/xray_security_policy/resource.tf`